### PR TITLE
feat: new grid

### PR DESCRIPTION
**Pull request by frabbit**

### CODE REVIEW

File: src/core/mixins/text.scss [hunk 1]
```diff
@@ -1,7 +0,0 @@
-@mixin hyphenate {
```
Comment (drinchev) [inline src/core/mixins/text.scss]: So those are all dead code? :D

File: src/core/globals/vars.scss [hunk 2]
```diff
@@ -68,6 +68,8 @@ $olt-link-hover-contrast: var-get('linkHoverContrast');
 $olt-grid-gutter: var-get('containerPadding');
 $olt-grid-columns: 12;
 
+$olt-v2-grid-gutter: 16px;
```
Comment (drinchev) [inline src/core/globals/vars.scss]: Let's put this into the V2Grid.scss.

File: src/core/globals/vars.scss [hunk 3]
```diff
@@ -68,6 +68,8 @@ $olt-link-hover-contrast: var-get('linkHoverContrast');
 $olt-grid-gutter: var-get('containerPadding');
 $olt-grid-columns: 12;
```
Comment (drinchev) [inline src/core/globals/vars.scss]: As well as those ;)

File: src/core/mixins/text.scss (hunk 1 above)
Comment (frabbit) [reply src/core/mixins/text.scss]: At least i didn't found a usage and it still compiles ;)

File: src/core/globals/vars.scss (hunk 2 above)
Comment (frabbit) [reply src/core/globals/vars.scss]: convinced ;)

File: src/core/globals/vars.scss (hunk 3 above)
Comment (frabbit) [reply src/core/globals/vars.scss]: done

**Pull request merged**